apps/authentication-service/src/middleware/auth0.middleware.ts

```diff
@@ -0,0 +1,10 @@
+import * as dotenv from "dotenv";
+import { auth } from "express-oauth2-jwt-bearer";
+
+dotenv.config();
+
+export const validateAccessToken = auth({
+  audience: process.env.AUTH0_AUDIENCE,
+  issuerBaseURL: `https://${process.env.AUTH0_DOMAIN}`,
```

Comment:
I'd probably have an environment.ts file somewhere that pulls everything from process.env